src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft/Build/Tasks/Windows/GenerateTemporaryTargetAssembly.cs

```diff
@@ -952,7 +952,7 @@ static XmlNode CreateImportProjectSdkNode(XmlDocument xmlProjectDoc, string proj
 
         private const string WPFTMP = "wpftmp";
 
-        private static readonly char[] _semicolonChar = new char[] { ';' };
+        private const char SemicolonChar = ';';
```

Comment:
If this is going into `string.Split(params char[])` then this will make it allocating array in every call, or did I miss something? Same with `ForwardSlashSeparator` below.

Reply:
Thanks, it will be marginally better this way.